pkg/virtualKubelet/forge/endpointslices.go

```diff
@@ -79,7 +78,10 @@ func RemoteEndpointSliceEndpoints(locals []discoveryv1beta1.Endpoint,
 
 		remote := discoveryv1beta1apply.Endpoint().
 			WithAddresses(translator(local.Addresses)...).WithConditions(conditions).
-			WithTopology(local.Topology).WithTopology(map[string]string{corev1.LabelHostname: nodename}).
+			WithTopology(local.Topology).WithTopology(map[string]string{
+			liqoconst.TypeLabel:       liqoconst.TypeNode,
+			liqoconst.RemoteClusterID: LocalCluster.ClusterID,
+		}).
```

Comment:
Why is this modified?

Reply:
Because the node name now depends on the cluster name, which in turn may vary across clusters. This change rewrites the query to be independent of names.

Reply:
Ok, but that is not the purpose of that field (https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#topology).

BTW, please do not mark as resolved conversations which are not effectively resolved.